src/GameManager.js

```diff
@@ -15,6 +15,10 @@ class GameManager {
   }
 
   endGame (guildId) {
+    Game.players.forEach(player => {
```

Comment:
That wouldn't work. You'd have to `this.getGame(guildId)` instead of using plain `Game`